README.md

```diff
@@ -142,16 +142,18 @@ information on its usage.
 
 ### Pick the modules you need
 
-| File | Minified + gzipped size | Runtime minified + gzipped size | Summary |
-|---|--:|--:|---|
-| globalize.js | 1.5KB | 1.0KB | [Core library](#core-module) |
-| globalize/currency.js | 2.6KB | 0.6KB | [Currency module](#currency-module) provides currency formatting and parsing |
-| globalize/date.js | 5.1KB | 3.8KB | [Date module](#date-module) provides date formatting and parsing |
-| globalize/message.js | 5.4KB | 0.7KB | [Message module](#message-module) provides ICU message format support |
-| globalize/number.js | 3.1KB | 1.8KB | [Number module](#number-module) provides number formatting and parsing |
-| globalize/plural.js | 2.3KB | 0.4KB | [Plural module](#plural-module) provides pluralization support |
-| globalize/relative-time.js | 0.8KB | 0.6KB | [Relative time module](#relative-time-module) provides relative time formatting support |
-| globalize/unit.js | 0.9KB | 0.5KB | [Unit module](#unit-module) provides unit formatting support |
+| File | Minified + gzipped size | Runtime minified + gzipped size | Dependencies (load in order) | Summary |
+|---|--:|--:|---|---|
+| globalize.js | 1.5KB | 1.0KB | Cldrjs & CLDR-data | [Core library](#core-module) |
+| globalize/currency.js | 2.6KB | 0.6KB | core, number.js & plural.js | [Currency module](#currency-module) provides currency formatting and parsing |
```

Comment:
"core" is inconsistent with the other references (which have the ".js" extension). Not sure if "globalize.js" is much better though.